WireDomain/Sources/WireDomain/Components/ClientSessionComponent.swift

```diff
@@ -316,7 +260,7 @@ public final class ClientSessionComponent {
         store: userLocalStore
     )
 
-    private lazy var pullSelfUserClientsSync = PullSelfUserClientsSync(
+    public lazy var pullSelfUserClientsSync = PullSelfUserClientsSync(
```

Comment:
Similar to the last comment, it's public because there's some other target that is using it.